recipes/capsula/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "capsula" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/capsula-{{ version }}.tar.gz
+  sha256: 34fe7d37e1e6a580785b842c53a27b4ba1a0e7bf74c721ce75833abb00c29d30
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - poetry-core
+    - pip
+  run:
+    - python >=3.8
+    - click >=8.1.3
+    - pydantic >=2.0.2
+    - py-cpuinfo >=9.0.0
+    - gitpython >=3.1.31
+    - tomli >=2.0.1
+    - typing-extensions >=4.7.1
+    - orjson >=3.9.10
+
+test:
+  imports:
+    - capsula
+  commands:
+    - pip check
+  requires:
+    - pip
+    - git
```

Comment:
Why is git required here?

Reply:
It is required to import `gitpython` (one of the dependencies) in the import check. [This CI](https://github.com/conda-forge/staged-recipes/runs/21193475384) failed because of this, for example.

After the CI failed I checked [the recipe for `gitpython`](https://github.com/conda-forge/gitpython-feedstock/blob/22871c8664ebc8d09b2adbb64b313f83a972b3c1/recipe/meta.yaml) and added git to my recipe as they did.